src/vcpkg/install.cpp

```diff
@@ -1209,13 +1221,19 @@ namespace vcpkg
             {
                 if (!result.is_user_requested_install()) continue;
                 auto bpgh = result.get_binary_paragraph();
-                assert(bpgh);
```

Comment:
I added this assert because it was not clear how this could ever be nullptr; can you add a comment describing how that happened? For example:

```
if (!bpgh) continue; // If --keep-going, packages may not be installed
```